app/Events/QueueMessageRequest.php

```diff
@@ -24,9 +25,10 @@ class QueueMessageRequest extends Event
      *
      * @return void
      */
-    public function __construct(Message $message)
+    public function __construct(Message $message, $sender)
```

Comment:
Now that I think about it, and not necessarily needed for merge, I wonder if we should just pass in an array or object that has all the info that we need about a message in it i.e the message, the sender, the list to send to. This way we only have to  grab one thing in the listener, and send the one array to the `Mail::send()` method, that can handle all the message configuration.